cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -409,22 +409,27 @@ private static <I extends ReactiveLimitsHolder & Injection<I>> double computeDef
 
     private static void writeShuntCompensators(Network network, Map<Terminal, String> mapTerminal2Id, Set<String> regulatingControlsWritten, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) throws XMLStreamException {
         for (ShuntCompensator s : network.getShuntCompensators()) {
-            double bPerSection = 0.0;
-            double gPerSection = Double.NaN;
-            if (s.getModelType().equals(ShuntCompensatorModelType.LINEAR)) {
-                bPerSection = ((ShuntCompensatorLinearModel) s.getModel()).getBPerSection();
-                gPerSection = ((ShuntCompensatorLinearModel) s.getModel()).getGPerSection();
-            }
-            String regulatingControlId = RegulatingControlEq.writeKindVoltage(s, exportedTerminalId(mapTerminal2Id, s.getRegulatingTerminal()), regulatingControlsWritten, cimNamespace, writer, context);
-            ShuntCompensatorEq.write(context.getNamingStrategy().getCgmesId(s), s.getNameOrId(), s.getSectionCount(), s.getMaximumSectionCount(), s.getTerminal().getVoltageLevel().getNominalV(), s.getModelType(), bPerSection, gPerSection, regulatingControlId,
-                    context.getNamingStrategy().getCgmesId(s.getTerminal().getVoltageLevel()), cimNamespace, writer, context);
-            if (s.getModelType().equals(ShuntCompensatorModelType.NON_LINEAR)) {
-                double b = 0.0;
-                double g = 0.0;
-                for (int section = 1; section <= s.getMaximumSectionCount(); section++) {
-                    ShuntCompensatorEq.writePoint(CgmesExportUtil.getUniqueId(), context.getNamingStrategy().getCgmesId(s), section, s.getB(section) - b, s.getG(section) - g, cimNamespace, writer, context);
-                    b = s.getB(section);
-                    g = s.getG(section);
+            if ("true".equals(s.getProperty(Conversion.PROPERTY_IS_EQUIVALENT_SHUNT))) {
+                // Must have been mapped to a linear shunt compensator with 1 section
+                EquivalentShuntEq.write(context.getNamingStrategy().getCgmesId(s), s.getNameOrId(), s.getG(), s.getB(), context.getNamingStrategy().getCgmesId(s.getTerminal().getVoltageLevel()), cimNamespace, writer, context);
```

Comment:
In shunt compensator in IIDM, we can change the section from 1 to 0. Even if it is an equivalent shunt because we don't have a dedicated modeling. In that case, we are going to export B = 0 and G = 0, which is for me a loose of information. In that case, the shunt compensator must be treated as disconnected.

Reply:
We propose to export the total G, B of the equivalent shunt in the EQ instance file, and declare it as disconnected in the SSH and SV output files.

Reply:
Ok !